### DISCUSSION TITLE
Cheat Engine?

### DISCUSSION THREAD

**Discussion by Flat** [SteamDeckHomebrew/decky-loader#134] [General]
The readme image seems to show a plugin for a cheat engine, is this simply a placeholder and one does not exist currently? I haven't been able to find anything regarding it and the image was committed quite a while ago.

**Comment by TrainDoctor** [comment]
Yes it's an old demo image. Cheat engine is windows exclusive and can't be nicely brought over to linux.

**Reply by rzabarazesh** [reply]
Cheat engine works perfectly fine with steamtinkerlaunch

**Comment by Flat** [comment]
This is true, I'm sure implementing something like scanmem would be much easier. Thanks for the answer.
